themes/bootstrap3/templates/feedback/form.phtml

```diff
@@ -3,10 +3,9 @@
   $form->prepare();
 
   $title = $form->getTitle();
-  $title = !empty($title)
-    ? $this->translate($title) : null;
+  $title = !empty($title) ? $this->translate($title) : null;
 
-  $formUrl = $this->url('feedback-form', ['id' => $this->formId]);
+  $formUrl = $this->url($this->formActionUrl ?? 'feedback-form', ['id' => $this->formId]);
```

Comment:
Where is formActionUrl set? Should this actually be `$this->formActionUrl ?? $this->url(...)` so you override the actual URL instead of the route name? If not, should the variable be named `formActionRoute` instead?

Reply:
hmm, actually setting the route in feedbackform yaml could be enough as everything else can be otbained from params and yaml, added a function for it now :)